HOW_TO_TEST.md

```diff
@@ -15,17 +15,26 @@ If the environment has already been setup, you can skip to [Send data](#send-dat
 
 - see https://serverless.com/framework/docs/providers/aws/guide/credentials/
 
-You mainly need to get your AWS keys and set them up by using:
+Firstly, you need to have a AWS account, and a pair of keys associated to it
+- we use a shared AWS account, with several users
+- once you have your user access, you need to [create your user's access key](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html)
+- that will give you an access key and a private key
+
+Then, you need to setup your pair of keys by typing in a terminal:
```

Comment:
"This will set 2 [environment variables](http://docs.aws.amazon.com/cli/latest/userguide/cli-environment.html) `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` for the session.